services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -122,8 +143,14 @@ def post(self, mine_guid):
 
         uploadedFiles = data.get('uploadedFiles', [])
 
+        permit_prefix = permit_no[0];
+        json_data = request.json
```

Comment:
I think `site_properties` should be added to the "arg parser", that way it can be accessed like the rest of the data?

Reply:
done